common/lib/mandrill_client/client.py

```diff
@@ -38,40 +46,66 @@ class MandrillClient(object):
     REFERRAL_FOLLOW_UP_EMAIL = 'referred-learner-follow-up'
     REFERRAL_SOCIAL_IMPACT_TOOLKIT = 'social-impact-toolkit'
     MINI_COURSE_ENROLMENT = 'mini-course-enrolment'
+    SEND_ACTION_PLAN = 'send-action-plan'
 
     def __init__(self):
         self.mandrill_client = mandrill.Mandrill(settings.MANDRILL_API_KEY)
 
-    def send_mail(self, template_name, user_email, context, attachments=[], subject=None):
+    def get_email_to(self, receiver_emails_string):
```

Comment:
Please rename the method with some better name may be `get_email_list` or get_receiver_emails` ..

Reply:
The purpose of the method is to convert emails string to a mail "to" format., thats why we named it to 'get_email_to'

Reply:
The name is confusing .. get_email_to isn't meaningful... 
in `to` we gave receiver's email .. So it is better to rename it to `get_receiver_emails`.

Reply:
okay, make sense  Renamed 👍